index.bs

```diff
@@ -4028,15 +4031,18 @@ readableStrategy)">new TransformStream(<var>transformer</var> = {}, <var>writabl
 
   <ul>
     <li><p><code>start(controller)</code> is called immediately, and is typically used to enqueue prefix <a>chunks</a>
-      that will be read from the <a>readable side</a> but don't depend on any writes to the <a>writable side</a>. If
-      this process is asynchronous, it can return a promise to signal success or failure.
+      using
+      {{TransformStreamDefaultController|controller}}.{{TransformStreamDefaultController/enqueue(chunk)|enqueue()}}.
+      Those chunks will be read from the <a>readable side</a> but don't depend on any writes to the <a>writable
+      side</a>. If this process is asynchronous, it can return a promise to signal success or failure.
 
     <li>
       <p><code>transform(chunk, controller)</code> is called when a new chunk originally written to the writable side is
       ready to be transformed. It can return a promise to signal success or failure of the transformation. The results
       of the transformation can be enqueued to the readable side using the
-      {{ReadableStreamDefaultController/enqueue(chunk)|controller.enqueue()}} method. This permits a single chunk
-      written to the writable side to result in zero or multiple chunks on the readable side.
+      {{TransformStreamDefaultController|controller}}.{{TransformStreamDefaultController/enqueue(chunk)|enqueue()}}
```

Comment:
What do you think about splitting the links to contain both the class definition as well as the method definition?

Reply:
Additionally, was it intentional that this linked to `ReadableStreamDefaultController.enqueue()` instead of `TransformStreamDefaultController.enqueue()`

Reply:
I'm 99% sure that linking to `ReadableStreamDefaultController.enqueue()` was a mistake here. Thanks for catching it.

Splitting the links looks okay to me, but I will defer to @domenic who knows much more about correct markup fro WhatWG standards.